flink/scripts/jobmanager-entrypoint.sh

```diff
@@ -0,0 +1,221 @@
+#! /bin/bash
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+set -e
+
+cobli_script="COBLI_ENTRYPOINT"
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+source ./common.sh
+
+submit_job() {
+  log_warn "> Submiting job..."
+  ensure_flink_config
+  /opt/flink/bin/standalone-job.sh "$@" 1>&2 &
```

Comment:
Is there a reason for throwing messages to stderr instead of stdout?

Reply:
In bash, stdout is the better option for function return